### DISCUSSION TITLE
how to set background color of selected line

### DISCUSSION THREAD

**Discussion by wothke** [NationalSecurityAgency/ghidra#3931] [Q&A] [answered]
By default the "Listing" view uses a light-blue background color to mark the line that is currently clicked on, or that corresponds to the line clicked on in the decompiler view.

I absolutely have to change that color since it is utterly useless on my monior (i.e. it is barely visible). Looking through the dozens of irrelevant color settings available in the tool settings I've been unable to find the one setting that would actually allow me to change that backgroud color.

Is there a respective settings available and I am just too dumb to find it?

**Comment by dragonmacher** [comment, marked as answer]
It is in options under `Listing Field -> Cursor -> Highlight Cursor Line Color`

**Reply by wothke** [reply]
thanks! :-)
